.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -183,17 +183,18 @@ jobs:
           github.event_name == 'workflow_dispatch' ||
           github.ref == 'refs/heads/master'
         with:
-          name: ${{ runner.os }}-wheels-${{ matrix.pl_backend }}-${{ matrix.arch }}.zip
+          name: ${{ runner.os }}-wheels-${{ matrix.pl_backend }}-${{ fromJson('{ "cp39-*":"py39","cp310-*":"py310","cp311-*":"py311","cp312-*":"py312" }')[matrix.cibw_build] }}-${{ matrix.arch }}.zip
```

Comment:
I am not well familiar with GitHub workflow but Is it not possible to add this to the matrix?

Reply:
If I recall correctly matrix evaluation happens at a different stage than the label processing here, meaning it would not generate the correct string. We can rely on the json processing in this way due to how the GA interpreter processes the overall YAML

Reply:
We don't need to add a new matrix variable as the info about the py version is embedded in `matrix.cibw_build`. I agree this is a bit weird though..